Formula/libsql-server.rb

```diff
@@ -0,0 +1,30 @@
+class LibsqlServer < Formula
+  desc "Server-mode binary for libSQL (sqld), an Open Contribution fork of SQLite"
+  homepage "https://libsql.org"
+  # pull from git tag to get submodules
+  url "https://github.com/libsql/sqld.git",
+      tag:      "v0.8.0",
+      revision: "4de673d04214a28188a45eefb60d2b2cc403859e"
+  license "MIT"
+
+  depends_on "rust" => :build
+  depends_on "protobuf"
```

Comment:
Looks like you have `openssl-sys` and `libgit2-sys` in your dependency tree. We probably want an dependency on `openssl` and `libgit2@1.5` here.

Reply:
Thank you! git should actually not be needed once we are able to build from source tar. Added openssl